sql-statements/sql-statement-select.md

```diff
@@ -159,6 +161,63 @@ mysql> SELECT AVG(s_quantity), COUNT(s_quantity) FROM stock;
 
 The above example uses data generated with `tiup bench tpcc prepare`. The first query shows the use of `TABLESAMPLE`.
 
+### SELECT ... INTO OUTFILE
+
+`SELECT ... INTO OUTFILE` is used to write the result of a query to a file. Column and line terminators can be specified to produce a specific output format.
+
+You can use the `Fields` and `Lines` parameters to specify how to handle the data format.
+
+- `FIELDS TERMINATED BY`: specifies the data delimiter.
+- `FIELDS ENCLOSED BY`: specifies the enclosing character of the data.
+- `LINES TERMINATED BY`: specifies the line terminator, if you want to end a line with a certain character.
+
+Below are some examples, first create a table and prepare some data:
+
+```sql
+mysql> create table t (a int, b varchar(10), c decimal(10,2));
+Query OK, 0 rows affected (0.02 sec)
+
+mysql> insert into t values (1, 'a', 1.1), (2, 'b', 2.2), (3, 'c', 3.3);
+Query OK, 3 rows affected (0.01 sec)
+```
+
+Then here are some `SELECT ... INTO OUTFILE` statements and their results.
+
+```sql
+mysql> select * from t into outfile '/tmp/tmp_file1';
+Query OK, 3 rows affected (0.00 sec)
+```
+
+```
+1       a       1.10
+2       b       2.20
+3       c       3.30
+```
```

Comment:
Maybe do something like ```mysql> ! cat /tmp/tmp_file1```  to make it more obvious that this isn't outputed by the client, but being written to the file?